### PR TITLE
Add new theme: powerline-wizard

### PR DESCRIPTION
**Pull request by tpCodeX**
Hi! I want to add a new theme 🫶

this is modified version of powerline-icon called powerline-wizard, hope you like 🧙

### CODE REVIEW

**Review by akinomyoga — CHANGES_REQUESTED** [review]
Thank you for your contribution.

The image needs to be created in a smaller size. The theme needs to be added to the list in `themes/THEMES.md`. Please check [the corresponding section in `CONTRIBUTING.md`](https://github.com/ohmybash/oh-my-bash/blob/master/CONTRIBUTING.md#new-theme)

**Comment by tpCodeX** [comment]
@akinomyoga Thanks for your response, this is my first time contributing to something. I just make the properly changes, so i'll do another PR. 
Thank you again :)
🧙

**Comment by akinomyoga** [comment]
Don't worry. Additional fixes are typical. Rather, merging without any fixes or adjustments is rare unless it is a trivial fix.

Please do not make another PR. The fixes must be made within this PR. Or otherwise, we wouldn't be able to track the discussion. If you push new commits to the branch that was used to create the PR, the pushed changed will be reflected in the PR page.

**Comment by akinomyoga** [comment]
I've checked the image. Thank you for updating the image. The new width is OK, but its height seems to be 110px. Have you read the section in `CONTRIBUTING.md`? If you haven't, could you read the corresponding section in `CONTRIBUTING.md`? I think the height of 110px would be actually fine if you intentionally made it smaller than the others, but it's not a typical size. You can make it larger if you want.

**Comment by tpCodeX** [comment]
Sorry!
It seems I pushed the wrong image, I'll correct it now. Thank you very much for responding so quickly!

**Comment by akinomyoga** [comment]
I took a diff between `powerline-icon/powerline-icon.base.sh` and `powerline-wizard/powerline-wizard.base.sh`:

```diff
--- themes/powerline-icon/powerline-icon.base.sh^I2024-09-01 07:55:30.833441609 +0900
+++ a.txt^I2024-09-01 07:55:42.897500275 +0900
@@ -27,17 +27,17 @@
       fi
       ;;
   esac
-  [[ -n "${user_info}" ]] && echo "🐧 ${user_info} `date +%X\ %D`|${color}|${secondary_color}"
+  [[ -n "${user_info}" ]] && echo "🧙 ${user_info} `date +%X\ %D`|${color}|${secondary_color}"
 }

 function __powerline_cwd_prompt {
-  echo "$(pwd | sed "s|^${HOME}|🏠|")|${CWD_THEME_PROMPT_COLOR}"
+  echo "$(pwd | sed "s|^${HOME}|🔮|")|${CWD_THEME_PROMPT_COLOR}"
 }

 function __powerline_last_status_prompt {
   if [[ "$1" -ne 0 ]]; then
-    echo "❌${1}|${LAST_STATUS_THEME_PROMPT_COLOR}"
+    echo "🔥${1}|${LAST_STATUS_THEME_PROMPT_COLOR}"
   else
-    echo "✅|${LAST_STATUS_THEME_PROMPT_COLOR_SUCCESS}"
+    echo "🌀|${LAST_STATUS_THEME_PROMPT_COLOR_SUCCESS}"
   fi
 }
```

The differences are essentially only the icons, so I'm now thinking of making the icons in `powerline-icon.base.sh` configurable through shell variables (such as `OMB_THEME_POWERLINE_ICON_{USER,HOME,FAILURE,SUCCESS}`) so that both `powerline-icon.theme.sh` and `powerline-wizard.theme.sh` can source a single common file `themes/powerline-icon/powerline-icon.base.sh`.

**Comment by tpCodeX** [comment]
It make sense to me, but I actually I don't know how to do it 😞

I'm new at linux & bash scripting :P

**Comment by akinomyoga** [comment]
OK, thank you for letting me know your thoughts! I'll commit the change.

**Comment by tpCodeX** [comment]
Thanks for taking my opinion into account, now I think the name "powerline-wizard" no longer makes sense, what do you suggest to do with it?

**Comment by akinomyoga** [comment]
I pushed two commits to the PR branch. Could you pull them and check the behavior?

File: themes/powerline-wizard/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# 🧙 Powerline-Wizard 🔮
```
Comment (akinomyoga) [inline themes/powerline-wizard/README.md:1]: The CI detects a trailing whitespace here.

```suggestion
# 🧙 Powerline-Wizard 🔮
```

**Comment by akinomyoga** [comment]
> Thanks for taking my opinion into account, now I think the name "powerline-wizard" no longer makes sense, what do you suggest to do with it?

Could you describe why you think the name doesn't make sense in more detail? The result is expected to be the same, so `powerline-wizard` seems fine.

**Comment by tpCodeX** [comment]
> > Thanks for taking my opinion into account, now I think the name "powerline-wizard" no longer makes sense, what do you suggest to do with it?
> 
> Could you describe why you think the name doesn't make sense in more detail? The result is expected to be the same, so `powerline-wizard` seems fine.

I had misunderstood how it worked, I have already reviewed your changes and now I understand correctly. Thank you very much for your time, I really appreciate your help.

**Comment by akinomyoga** [comment]
Thank you for your clarification!

**Review by akinomyoga — APPROVED** [review]

**Pull request merged**